src/platform.ts

```diff
@@ -117,6 +117,9 @@ export class VieramaticPlatform implements DynamicPlatformPlugin {
         return;
       }
     }
+    if (device.friendlyName) {
```

Comment:
`if (device.friendlyName) {` -> `if (device.friendlyName?) {`  [just to make it more idiomatic]

Reply:
Linter keeps arguing with me over `if (device.friendlyName?) { tv.specs.friendlyName = device.friendlyName }` and `npm run build` throws an `Expression expected` error. 

Using a default fallback of `tv.specs.friendlyName = device.friendlyName ?? tv.specs.friendlyName` achieves the same thing.

Reply:
never mind. was in overclock mode. `tv.specs.friendlyName = device.friendlyName ?? tv.specs.friendlyName` is nicer actually. :-) since i just merged this. if you wish just do a nano pr with this mod :-) and i'll merge it in a snap. 

Thanks again for your patience && hard work!